From 95781c85283d7ef1d92fe4632fc395c15a7a3d68 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:30 -0400
Subject: [scsi] qla2xxx: Re-sync module parameter descriptions with the code

Message-id: <1408391817-6389-43-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87952
O-Subject: [PATCH RHEL 7.1 e-stor 42/69] qla2xxx: Re-sync module parameter descriptions with the code
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: "Steven J. Magnani" <steve.magnani@digidescorp.com>

scsi commit id b97f5d0b03034c1a99f2a4f8948def282ab64c7e

Fix module parameter descriptions mentioning default values that no longer
match the code.

Signed-off-by: Steven J. Magnani <steve@digidescorp.com>
Acked-by: Chad Dupuis <chad.dupuis@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_os.c b/drivers/scsi/qla2xxx/qla_os.c
index e199b05..48827cf 100644
--- a/drivers/scsi/qla2xxx/qla_os.c
+++ b/drivers/scsi/qla2xxx/qla_os.c
@@ -120,15 +120,17 @@ MODULE_PARM_DESC(ql2xmaxqdepth,
 int ql2xenabledif = 2;
 module_param(ql2xenabledif, int, S_IRUGO);
 MODULE_PARM_DESC(ql2xenabledif,
-  " Enable T10-CRC-DIF "
-  " Default is 0 - No DIF Support. 1 - Enable it"
-  ", 2 - Enable DIF for all types, except Type 0.");
+  " Enable T10-CRC-DIF:\n"
+  " Default is 2.\n"
+  "  0 -- No DIF Support\n"
+  "  1 -- Enable DIF for all types\n"
+  "  2 -- Enable DIF for all types, except Type 0.\n");
 
 int ql2xenablehba_err_chk = 2;
 module_param(ql2xenablehba_err_chk, int, S_IRUGO|S_IWUSR);
 MODULE_PARM_DESC(ql2xenablehba_err_chk,
   " Enable T10-CRC-DIF Error isolation by HBA:\n"
-  " Default is 1.\n"
+  " Default is 2.\n"
   "  0 -- Error isolation disabled\n"
   "  1 -- Error isolation enabled only for DIX Type 0\n"
   "  2 -- Error isolation enabled for all Types\n");
-- 
1.7.1